_data/projects/gkv.yml

```diff
@@ -0,0 +1,10 @@
+name: Gkv
+desc: An open source, key value store for small, simple projects leveraging git as the storage engine
+site: https://github.com/ybur-yug/gkv
+tags:
+  - database
+  - ruby
+  - git
+upforgrabs:
+  name: Gkv
```

Comment:
Similarly, this is the label name associated with the curated issues - `help wanted` is the example from the default repository.